src/Illuminate/Database/Schema/Grammars/MySqlGrammar.php

```diff
@@ -560,6 +560,17 @@ protected function typeBinary(Fluent $column)
     }
 
     /**
+     * Create the column definition for a uuid type.
+     *
+     * @param  \Illuminate\Support\Fluent  $column
+     * @return string
+     */
+    protected function typeUuid(Fluent $column)
+    {
+        return 'char(36)';
```

Comment:
I know. I spent quite some time deliberating on this while preparing the PR. Technically we can define it as BINARY(16) for MySQL and BLOB for SQLite. But that would mean that you'd be no longer able to simply use a string representation i.e. 'a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11' for assigning values and probably more importantly using them in queries in case of MySQL/SQLite.

I'd say we better introduce an absent `binaryString` and `binaryStringVarying` types to the grammar so that seasoned users can take advantage of BINARY SQL type to store uuid values.

Also up to some threshold (tens of thousands of rows) the performance degradation will be insignificant. But the size wise difference of course will be considerable.